File: transektcount/src/main/java/com/wmstein/transektcount/widgets/AddAlertWidget.java
package com.wmstein.transektcount.widgets;

import android.content.Context;
import android.util.AttributeSet;
import android.view.LayoutInflater;
import android.widget.LinearLayout;

import com.wmstein.transektcount.R;

import java.util.Objects;

/****************************************************
 * Created by milo on 01/06/2014.
 * Adopted for TransektCount by wmstein on 18.02.2016
 * Last edited on 2019-02-12
 */
public class AddAlertWidget extends LinearLayout
{

    public AddAlertWidget(Context context, AttributeSet attrs)
    {
        super(context, attrs);

        LayoutInflater inflater = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
        Objects.requireNonNull(inflater).inflate(R.layout.widget_add_alert, this, true);
    }

}

File: transektcount/src/main/java/com/wmstein/transektcount/widgets/AddAlertWidget.kt
package com.wmstein.transektcount.widgets

import android.content.Context
import android.util.AttributeSet
import android.view.LayoutInflater
import android.widget.LinearLayout
import com.wmstein.transektcount.R
import java.util.Objects

/******************************************************
 * Created by milo on 01/06/2014.
 * AddAlertWidget.kt is used by CountOptionsActivity.kt.
 * Adopted for TransektCount by wmstein on 18.02.2016,
 * last edited in Java on 2019-02-12,
 * converted to Kotlin on 2023-06-26,
 * last edited on 2025-11-15
 */
class AddAlertWidget(context: Context, attrs: AttributeSet?) : LinearLayout(context, attrs) {
    init {
        val inflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
        Objects.requireNonNull(inflater).inflate(R.layout.widget_add_alert, this, true)
    }

}
